registration/backends/enhanced/urls.py

```diff
@@ -0,0 +1,73 @@
+"""
```

Comment:
@dicato do you have an opinion between sharing URLs between backends? All of these URLs are the same from the `default` backend, except the `approve` related ones. Should we just append to the `default` urls or leave this file as is?